### ISSUE TITLE
0.3.0: 'TestCaseFunction' object has no attribute 'get_closest_marker'

### ISSUE THREAD

**Issue by kevinkjt2000** [miketheman/pytest-socket#11]
I noticed an error from the 0.3.0 update on one of my CI jobs:
```
    def pytest_runtest_setup(item):
>       mark_restrictions = item.get_closest_marker('allow_hosts')
E       AttributeError: 'TestCaseFunction' object has no attribute 'get_closest_marker'
../../../virtualenv/python3.6.3/lib/python3.6/site-packages/pytest_socket.py:90: AttributeError
```
https://github.com/kevinkjt2000/bowser/pull/165
https://travis-ci.org/kevinkjt2000/bowser/builds/404137945

Are there breaking changes that would cause this?

**Comment by kevinkjt2000** [comment]
If that is the case, bumping the minimum version of pytest would be helpful.  The CI build that I mentioned always does `pip check` to verify the dependencies are compatible. 
 Unfortunately, I cannot upgrade pytest yet (https://github.com/kevinkjt2000/bowser/pull/161).

**Comment by miketheman** [comment]
The dep update has been released with #12 in v0.3.1.
Thanks for reporting @kevinkjt2000 and for fixing @sadams !
